src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerFeatureOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Razor.Workspaces
+{
+    internal abstract class LanguageServerFeatureOptions
```

Comment:
Added to CodeAnalysis to avoid: https://github.com/dotnet/aspnetcore/issues/24385